hw/ip/prim/dv/prim_present/prim_present_sim.core

```diff
@@ -0,0 +1,27 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:prim_present_sim:0.1"
+description: "PRESENT block cipher DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:prim:all
+    file_type: systemVerilogSource
+
+  files_dv:
+    depend:
+      - lowrisc:dv:dv_utils
```

Comment:
Do you use this lib in your tb? If not, maybe we can delete this line?

Reply:
good catch! I don't really use anything from this lib in the testbench, I'll remove this.